executor/aggregate.go

```diff
@@ -603,10 +605,14 @@ func (e *HashAggExec) parallelExec(ctx context.Context, chk *chunk.Chunk) error
 		}
 	})
 
+	if e.executed {
```

Comment:
should we put this check to the `Next()` function?

Reply:
`unparallelExec` does not have the same problem, we may keep this check in `parallelExec` for now.